R/create_blogpost.R

```diff
@@ -15,12 +15,12 @@ c(
 
 # Fill in the info, e.g.:
 create_post(
-  post_name = "iscr_conference", # needs to be character vector (length 1)
-  post_date = "2024-02-17", # needs to be character vector (length 1)
-  description = "", # you can fill the description in later as well
-  author = c("Pooja Kumari"), # one or more authors are permitted
+  post_name = "Inside the Pharmaverse", # needs to be character vector (length 1)
+  post_date = "2024-03-04", # needs to be character vector (length 1)
+  description = "A short blog to help the Pharmaverse community understand how Pharmaverse is governed", # you can fill the description in later as well
+  author = c("Michael Rimler", "Ross Farrugia", "Mike Stackhouse", "Sumesh Kalappurakal", "Ari Siggaard Knoph"), # one or more authors are permitted
   cover_image = "pharmaverse", # chose one of the above (see line 8)
-  tags = c("submission", "ADaMs", "conferences", "admiral", "xportr", "metatools", "metacore") # chose (multiple) from line 10
+  tags = c("community") # chose (multiple) from line 10
```

Comment:
@rossfarrugia : I was under the understanding that I selected from Line 10 - which is the source list. Then, when the function is executed, the header is automatically populated (which it was). I did not edit line 10.

Reply:
you did the right thing in what you ran to use this function to create your blog folder template, but you didn't need to push this bit and overwrite this example function call in the repo itself. as this is given as a starter to help people adapt and then run to create their blog starting point. i'll push a commit to fix this now so that you know what i mean.

Reply:
just updated - not sure it really matters though tbh, as i saw others have updated this previously in some PRs before too